archive/o/opa/README.md

```diff
@@ -0,0 +1,25 @@
+# Sample Programs in Opa
+
+Welcome to Sample Programs in Opa!
+
+## Sample Programs
+
+- Hello World in Opa
+
+## Fun Facts
+
+- Debut: 2011
+- Developer: MLState
+- Typing: Static
+
+## How to run
```

Comment:
Hey! I'm a fan of this section. That said, I'm curious how this section will scale with more programs in the folder. Would you be interested in removing this section and instead writing a full article for it?

Reply:
Actually, I just noticed you're not interested in writing the article. How about we copy this section into an issue, so whoever writes the article can then include it.